src/types/platforms-types/catalog.ts

```diff
@@ -1,12 +1,13 @@
 import _ from 'lodash';
 
-import { ethereumTrackId, ethereumId, slugify } from '../../utils/identifiers';
+import { evmTrackId, evmId, slugify } from '../../utils/identifiers';
 import { ArtistProfile } from '../artist';
+import { ChainId } from '../chain';
 import { MapNFTsToTrackIds, MapTrack } from '../mapping';
 import { NFT } from '../nft';
 
-const mapAPITrackToArtistID = (apiTrack: any): string => {
-  return ethereumId(apiTrack.artist.id);
+const mapAPITrackToArtistID = (chainId: ChainId, apiTrack: any): string => {
+  return evmId(chainId, apiTrack.artist.id);
```

Comment:
should this be hardcoded to ethereum, since it's an artist ID? and use artistId() instead of evmId()?